### ISSUE TITLE
Text alignment option does nothing

### ISSUE THREAD

**Issue by snezhniylis** [foobnix/LibreraReader#929]
No matter which option I choose, it seems to be stuck on "Justified".

<details>
  <summary>Screenshots</summary>
  
  ![Screenshot_20220914-170156_Librera_PRO](https://user-images.githubusercontent.com/85341530/190176210-19ba1d20-0114-423b-8e16-1ab8a6b1ed75.png)
![Screenshot_20220914-170150_Librera_PRO](https://user-images.githubusercontent.com/85341530/190176252-72ed423d-c3c5-431a-9093-8a7208bd4ebd.png)
  
</details>

This has been tested on numerous different `epub`s from my library.

Librera version: `8.6.27-fdroid`

**Comment by foobnix** [comment]
Please send the book sample.
It may be the document CSS style "Justified" in this book.
Librera should override it if you change from "Document+User-defined" to "User-defined"  styles, please check.

**Comment by snezhniylis** [comment]
You were right. I went through my epub stash in Calibre and virtually all of these books have `text-align: justify` defined in some form. Turns out, KOReader always overwrote those by user-set values.

Switching to User-defined styles + adding  `p[class|="center"] { text-align: center !important;}` to user CSS makes it more or less alright. Thank you.

The issue is resolved, I presume? I can still attach a sample if needed.
